tutorials/v7/forest/fst001_staff.C

```diff
@@ -15,10 +15,6 @@
 // Functionality, interface, and data format is still subject to changes.
 // Do not use for real data!
 
-// The following line should disappear in a future version of RForest, when
-// the common template specializations of RField are part of the LinkDef.h
-R__LOAD_LIBRARY(ROOTForest)
```

Comment:
It did not yet figure out why this is needed sometimes, but I got otherwise error messages on missing `RField<unsigned, void>` symbol.  It might be better to leave it in with the comment so that the tutorial reliably works.

Reply:
I tried on osx and it worked and one of the goals of this pr was to re-try everything on all platforms.
Is there a symbol we can rely on which is present in all tutorials and select it in order to use it as a hook to make sure autoloading always happen?
I am for sure biased but when I see ROOT autoloading not working, I feel we are missing an oopportunity to show one of the most powerful features of the toolkit ;-)

Reply:
I narrowed it down to the use of `model->MakeField<unsigned int>` resp. `model->MakeField<std::uint32_t>`. If the tutorial only instead uses `model->MakeField<int>` or `model->MakeField<std::int32_t>`, it works without `R__LOAD_LIBRARY` on my machine. Still puzzled, but maybe for the moment we can remove `R__LOAD_LIBRARY` and turn the flags field into an int.

Reply:
I can do it.

Reply:
the issue I think is that a template is instantiated with a type which is not selected in the Linkdef.

Reply:
I think RFieldRoot would do the job.
Shall I add it to the Linkdef And merge or the opposite order?

Reply:
Hm, nothing I add to LinkDef.h makes it work on my laptop.  However, using `MakeField<std::uin64_t>` (64bit instead of 32bit) does autoload correctly.  Could it simply be a problem of the arch gcc 8.3? You certainly have more experience than me, please feel free to merge the PR.

Reply:
We know of a case where we need `R__LOAD_LIBRARY`. Until we can remove it I'd prefer to leave `R__LOAD_LIBRARY` to have working tutorials, everywhere.